shared/ff_util/include/ff_util/ff_timer.h

```diff
@@ -48,26 +48,30 @@ class FreeFlyerTimer {
   // Destructor
   ~FreeFlyerTimer() {}
 
-  void createTimer(double period_sec,
+  void createTimer(double period,
                    TimerCallbackType cb_timer,
                    rclcpp::Node::SharedPtr node,
                    bool one_shot = false,
                    bool auto_start = true) {
-    createTimer(std::make_shared<rclcpp::Duration>(period_sec, 0),
+    int period_sec = std::floor(period);
```

Comment:
@kbrowne15 I was getting a really high frequency and I think it's because it was rounding the seconds to zero, this fixed it

Reply:
Okay, thanks for fixing it.